Presentations/Week01/R4DS Ch 1-2 - Morris.Rmd

```diff
@@ -0,0 +1,189 @@
+---

```

Comment:
Looks awesome, Luke! 
We've got a second cohort (Europe/Africa) coming right down the barrel at us, so if you could add your stuff to a subfolder in Week01 that would be great (ie Presentations/Week01/Cohort1)